OpenOversight/app/utils/constants.py

```diff
@@ -60,3 +60,57 @@
 FLASH_MSG_PERMANENT_REDIRECT = (
     "This page's address has changed, please update your bookmark!"
 )
+
+# Map Path Constants
+MAP_PATHS = {
+    "FL": "M792.3,457.6l2.4,7.6,3.9,10.1,5.5,9.8,3.9,6.6,5,5.7,4.2,3.9,1.7,3-1.1,1.4-.8,1.4,3,7.7,3,3,2.7,5.5,3.8,6,4.7,8.5,1.4,7.9.5,12.5.6,1.9-.3,3.5-2.5,1.4.3,2-.6,2,.3,2.5.5,2-2.8,3.3-3.2,1.6-4.1.2-1.6,1.7-2.5,1-1.4-.5-1.1-1-.3-3-.8-3.5-3.5-5.4-3.8-2.4-4.1-.3-.8,1.4-3.2-4.6-.6-3.8-2.7-4.2-1.9-1.1-1.7,2.2-1.9-.3-2.2-5.2-3-4.1-3-5.5-2.7-3.2-3.8-3.9,2.2-2.5,3.3-5.7-.2-1.7-4.7-1-1.7.6.3.6,2.7,1-1.6,4.7-.8.5-1.9-4.2-1.4-5-.3-2.8,1.6-4.9v-9.9l-3.2-3.9-1.4-3.2-5.4-1.4-2-.6-1.7-2.7-3.5-1.7-1.1-3.5-2.8-1-2.5-3.9-4.4-1.6-3-1.6h-2.6l-4.2.8-.2,2,.8,1-.5,1.1-3.2-.2-3.9,3.8-3.8,2h-4.1l-3.3,1.4-.3-2.8-1.7-2-3-1.1-1.7-1.6-8.4-4.1-7.9-1.9-4.6.6-6.3.5-6.3,2.2-3.7.6-.2-8.3-2.7-2-1.9-1.9.3-3.2,10.6-1.4,26.6-3,7.1-.6,5.6.3,2.7,4.1,1.6,1.6,8.4.5,11.3-.6,22.4-1.4,5.6-.7,5.3.2.4,3,2.3.8.2-4.8-1.6-4.4,1.4-1.5,5.8.5,5.6.4.2.2h0ZM805.4,595.6l2.5-.6,1.4-.2,1.6-2.4,2.4-1.7,1.4.5,1.8.3.4,1.1-3.6,1.2-4.4,1.6-2.4,1.3-.9-1.1h-.2ZM819.5,590.4l1.3,1.1,2.8-2.2,5.5-4.4,3.9-4.1,2.6-6.9,1-1.8.2-3.5-.7.5-1,2.9-1.6,4.8-3.3,5.5-4.6,4.4-3.5,2-2.5,1.6h-.1Z",
```

Comment:
I'm not sure how I feel about this. It does feel a bit messy but the equivalent (putting the data in a JSON/CSV file and importing it as a resource) seems comparable in terms of the format but we'd also need to add additional handling to import this data as a resource.

Happy to move this into another file if that would be preferable.

Reply:
I get why you'd put this here, and I have similar feelings about it in this file. Since it is static data, I'd be cool with putting it in a JSON file by itself. Presuming we don't mount the whole `static` directory to the frontend, we could put it in a `misc` category folder.

Reply:
Updated to import the map data as a json file